utils/python/CIME/SystemTests/err.py

```diff
@@ -29,6 +29,13 @@ def run_phase(self):
             rundir = self._case.get_value("RUNDIR")
             logger.info("staging files from archive %s" % dout_s_root)
             for item in glob.glob(os.path.join(dout_s_root, "*", "hist", "*base")):
+                # NOTE - this loop doesn't do anything because the
```

Comment:
@fischer-ncar and I decided to leave this comment in place as a reminder
of what needs to happen in order to address issue #416.